File: Java/Car.java
class Car{
    Integer id;
    String license;
    Account drive;
    Integer passegenger;


        public Car(String license, Account drive){
            this.license = license;
            this.drive = drive;
        }
    void printDataCar(){
        System.out.println("License: " + license + "Name driver: " + drive.name);
    }
}

File: PHP/Car.php
<?php

    class Car{
        Public $id;
        public$license;
        public $driver;
        public $passenger;

        public function __construc($license, driver){
            $this->license = $license;
            this->driver = $driver;
        }

        public function printDataCar(){
            echo "Licencia:  $this->license Driver: ".this->driver->name;
        }
    }
?>